### DISCUSSION TITLE
Visual FoxPro 9 Installation on Windows 10

### DISCUSSION THREAD

**Discussion by JayVFP9** [community/community#46779] [Other Feature Feedback, Questions, & Ideas]
Good afternoon,

To quote an amazing album.... "Is there anybody out there?".

I know this may sound a bit archaic but, I need some help installing Visual FoxPro 9 on a new Windows 10 computer. I had the software installed on a previous computer and it worked fine. I read an article by a person named "w00dy" and it was helpful, but I still had some questions. Is there anyone around that I can communicate with to help me with the installation steps and identify any problems I might encounter?

I would greatly appreciate any help someone can provide to help me get one more step down the path.

With my best regards,

Jay

JayVFP9

**Comment by ossanna16** [comment]
Hi @JayVFP9,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Visual FoxPro 9. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by BarcodeM** [comment]
perhaps  Microsoft Visual FoxPro ODBC driver is necessary  for Installation.
